experimental/rust/image/Dockerfile-stack

```diff
@@ -20,6 +20,9 @@ ENV APPSODY_DEBUG="cargo build && lldb-server platform --listen '*:1234' --min-g
 ENV APPSODY_DEBUG_ON_CHANGE="$APPSODY_DEBUG"
 ENV APPSODY_DEBUG_KILL=true
 
+RUN useradd rust
+USER rust
+
 COPY ./LICENSE /licenses
 COPY ./project /project
 COPY ./config /config
```

Comment:
Should we also add `--chown` to the copy command?

Reply:
can do